packages/components/src/styles/prose/prose.test.stories.ts

```diff
@@ -0,0 +1,740 @@
+  </p>
+  <h3>Sometimes I even use <code>code</code> in headings</h3>
+  <p>Even though it's probably a bad idea, and historically I've had a hard time making it look good.</p>
+  <p>
+    Another thing I've done in the past is put a <code>code</code> tag inside of a link, like if I wanted to tell you
+    about the <a href="https://github.com/tailwindcss/docs"><code>tailwindcss/docs</code></a> repository.
+  </p>
+  <h4>We haven't used an <code>h4</code> yet</h4>
+  <h5>And a <code>h5</code>neither</h5>
+  <p>
+    But now we have. Please don't use <code>h6</code> in your content, Medium only supports two heading levels for a
+    reason, you animals. We honestly considered using a <code>before</code> pseudo-element to scream at you if you use
+    <code>h6</code>.
+  </p>
+  <p>
+    We don't style <code>h6</code> at all out of the box because <code>h5</code> elements are already so small that they
+    are the same size as the body copy. What are we supposed to do with an <code>h5</code>, make it smaller than the
+    body copy? No thanks.
+  </p>
+  <h3>We still need to think about stacked headings though.</h3>
+  <h4>Let's make sure we don't screw that up with <code>h4</code> elements, either.</h4>
+  <p>Phew, with any luck we have styled the headings above this text and they look pretty good.</p>
+  <p>
+    Let's add a closing paragraph here so things end with a decently sized block of text. I can't explain why I want
+    things to end that way but I have to assume it's because I think things will look weird or unbalanced if there is a
+    heading too close to the end of the document.
+  </p>
+  <p>What I've written here is probably long enough, but adding this final sentence can't hurt.</p>`;
+
+export const Default = {
+  name: 'Default',
+  render: (args: { [k: string]: any }) => {
+    return generateTemplate({
+      options: {
+        templateContent: html` <div
+          class="sd-container ${args['sd-prose--inverted-attr'] === true
+            ? 'sd-container--variant-primary'
+            : 'sd-container--variant-white'}"
+        >
+          <div class="%CLASSES%">${args['default-slot'] || content}</div>
+        </div>`
+      },
+      args
+    });
+  }
+};
+
+export const Inverted = {
+  name: 'Inverted',
+  render: (args: { [k: string]: any }) => {
+    return generateTemplate({
+      axis: {
+        x: [{ type: 'attribute', name: 'sd-prose--inverted', values: [false, true] }]
+      },
+      options: {
+        templateContent: html` <div class="p-4">
+          <style>
+            .sd-prose--inverted table {
+              opacity: 0;
+            }
+          </style>
+          <div class="%CLASSES%">${args['default-slot'] || content}</div>
+        </div>`,
+        templateBackgrounds: { alternate: 'x', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
+      },
+      args
+    });
+  }
+};
+
+export const Samples = {
```

Comment:
Should these samples be moved to the templates @coraliefeil?

Reply:
Fixed according to figma